### DISCUSSION TITLE
Allow wb navigation between created records of particular table

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5608] [Feature Requests]
For example, if you did not expect a data set upload to create new agents, it would be useful to navigate between the cells that created new agents.

<img width="262" alt="Screen Shot 2022-07-22 at 21 38 42" src="https://user-images.githubusercontent.com/40512816/180587512-4beab113-a4a1-470f-b78c-05a76c3380f1.png">

You can manually check each blue cell to see if it was responsible for creating a new agent, but that is infeasible for all but the smallest data sets

Suggested by @grantfitzsimmons

**Comment by maxpatiiuk** [comment]
Actually, there is a workaround:
Make "Agents" table must match for that data set, and then all cells that would cause new agent to be created would cause an error.

**Comment by maxpatiiuk** [comment]
Adding UI for https://github.com/specify/specify7/pull/1857 would fix this issue

**Comment by CarolineDenis** [comment]
@maxpatiiuk was this fixed with the merge of #1857?

**Comment by maxpatiiuk** [comment]
Unfortunately not yet.

That pull request added a way to create a Record Set from workbench upload results on demand

However, at the moment the API supports creating Record Sets from the base table records only

It needs to be extended to allow passing in a table name too (i.e, in a data set that uploaded Collection Objects, you want to create a record set of all newly created Localities)

Then, the upload results dialog would have to be updated to use that endpoint
